teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>Link to TIP3P and GAFF references, maybe?</p><p> </p><p>Capitalize Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Didn't find anything suitable about tip3p, especially not in context with amber.  Added a link to the wikipedia article about water models which includes a section for tip3p. Better recommendations are welcome</p>

Reply:
I think this is the canonical citation: https://aip.scitation.org/doi/10.1063/1.445869